frontend/components/App/Participants/Participant.tsx

```diff
@@ -51,18 +50,21 @@ const ParticipantComponent: React.FC<{ participant: Participant; speaker?: boole
       <div className="social">
         {twitter ? (
           <Link href={twitter} passHref>
-            <GAButton
+            <a
+              onClick={() => {
+                pushEventDataLayer({
+                  action: 'Twitter',
+                  category: 'Participants',
+                  label: window.location.href
+                });
+              }}
+              href={twitter}
               target="_blank"
               rel="noreferrer"
-              className="icon"
-              variant="link"
-              event={{
-                action: 'Twitter',
-                category: 'Participants',
-                label: window.location.href
-              }}>
+              className="icon">
+              {' '}
```

Comment:
I think this is not necessary

Reply:
Actually this was Prettier but when I delete it, it wont appear anymore, strange.